tests/ci.rs

```diff
@@ -1,69 +1,10 @@
-use gh_workflow::*;
-use release_plz::Release;
-use toolchain::Toolchain;
+use gh_workflow_tailcall::*;
 
 #[test]
-fn ci() {
-    let flags = RustFlags::deny("warnings");
+fn generate_ci_workflow() {
+    let workflow = Workflow::default()
+        .auto_release(true)
+        .name("Build and Test".into());
```

Comment:
```suggestion
```

`name` doesn't need to be set.

Reply:
Okay , I will remove that and push it within some time